### PR TITLE
Dev coords

### PR DESCRIPTION
**Pull request by leaspark**
Lookup based on coordinates, rather than city. Minor refactoring, included ticketmaster in the data set as well.

### CODE REVIEW

File: classes/sourceconstructor.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ class SourceConstructor {
 
 	async grab(req, res){
 		var requestURL = this.getRequestUrl(req); 
+		console.log(requestURL)
```
Comment (mavysavydav) [inline classes/sourceconstructor.js]: should remove this
```suggestion
```

File: classes/sourceconstructor.js [hunk 2]
```diff
@@ -50,7 +50,11 @@ class SourceConstructor {
 		for (const [key, value] of Object.entries(sourceParams)) {
 			requestURL += key + "=" + value + '&';
 		}
+		if(this.apiName == "Eventbrite"){
+			requestURL += "expand=venue&"
+		}
 
+		console.log("the request is: ", requestURL)
```
Comment (mavysavydav) [inline classes/sourceconstructor.js]:
```suggestion
```

File: classes/sourceconstructor.js [hunk 3]
```diff
@@ -50,7 +50,11 @@ class SourceConstructor {
 		for (const [key, value] of Object.entries(sourceParams)) {
 			requestURL += key + "=" + value + '&';
 		}
+		if(this.apiName == "Eventbrite"){
```
Comment (mavysavydav) [inline classes/sourceconstructor.js:52]: @ssgrover should this method be overwritten in the eventbrite.js file to include this chunk of code? That might not be as clean as this tho. Not sure

File: service/events.js [hunk 4]
```diff
@@ -46,6 +44,8 @@ var events = {
    grab: function(req, res) {
       var formattedEvents = []; 
 
+      console.log("I am here!")
```
Comment (mavysavydav) [inline service/events.js]:
```suggestion
```

File: service/events.js [hunk 5]
```diff
@@ -56,6 +56,7 @@ var events = {
       Promise.all(requests).then(function (returnvals){
         for(var i = 0; i < sourceObjects.length; i++){
           formattedEvents.push(sourceObjects[i].formatEvents(returnvals[i]))
+          console.log("finished one.")
```
Comment (mavysavydav) [inline service/events.js]:
```suggestion
```

File: sources/eventbrite.js [hunk 6]
```diff
@@ -19,7 +25,9 @@ var eventbrite = new SourceConstructor({
 					    ticketUrl: res["events"][i]["url"],
 					    id: res["events"][i]["id"],
 					    tags: ["eventbrite"],
-					    imageUrl: res["events"][i]["logo"]["url"]
+					    //imageUrl: res["events"][i]["logo"]["url"],
```
Comment (mavysavydav) [inline sources/eventbrite.js:28]: is this supposed to be commented out?

**Comment by mavysavydav** [comment]
great stuff, made a few comments. one theme - remember to remove console.log statements!!

File: classes/sourceconstructor.js (hunk 3 above)
Comment (ssgrover) [reply classes/sourceconstructor.js:52]: Yes sourceconstructor shouldn't have any mention of other APIs. Better to move this to an override in eventbrite.js

**Review by ssgrover — CHANGES_REQUESTED** [review]

**Pull request merged**